extensions/parquet/base/src/main/java/io/deephaven/parquet/base/PageMaterializer.java

```diff
@@ -3,16 +3,11 @@
 //
 package io.deephaven.parquet.base;
 
-import io.deephaven.parquet.base.materializers.*;
 import org.apache.parquet.schema.LogicalTypeAnnotation;
-import org.apache.parquet.schema.PrimitiveType;
-import org.jetbrains.annotations.NotNull;
 
 import java.math.BigDecimal;
 import java.math.BigInteger;
 
-import static org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName.BINARY;
-
 public interface PageMaterializer {
 
     /**
```

Comment:
You pointed out that `resolveDecimalLogicalType` can move back to `ParquetSchemaReader`.